site/src/Controller/NotificationController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Notification;
+use Mgilet\NotificationBundle\NotifiableInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Mgilet\NotificationBundle\Manager\NotificationManager;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class NotificationController
+ */
+class NotificationController extends AbstractController
+{
+    /**
+     * @var NotificationManager
+     */
+    protected $notificationManager;
+
+    /**
+     * NotificationController constructor.
+     *
+     * @param NotificationManager $notificationManager
+     */
+    public function __construct(NotificationManager $notificationManager)
+    {
+        $this->notificationManager = $notificationManager;
+    }
+
+    /**
+     * Mark a Notification as seen
+     *
+     * @Route("/notification/seen", name="notification_mark_seen")
+     *
+     * @Method("POST")
+     *
+     * @param Request $request
+     *
+     * @throws \Doctrine\ORM\NonUniqueResultException
+     * @throws \Doctrine\ORM\EntityNotFoundException
+     * @throws \Exception
+     *
+     * @return JsonResponse
+     */
+    public function markAsSeenAction(Request $request)
+    {
+        $notification = $request->get('notification');
+        $notifiable = $request->get('notifiable');
+        $this->notificationManager->markAsSeen(
+            $this->notificationManager->getNotifiableInterface($this->notificationManager->getNotifiableEntityById($notifiable)),
+            $this->notificationManager->getNotification($notification),
+            true
+        );
+        $count = $this->notificationManager->getUnseenNotificationCount(
+            $this->notificationManager->getNotifiableInterface($this->notificationManager->getNotifiableEntityById($notifiable))
+        );
+
+        return new JsonResponse($count);
+    }
+
+    /**
+     * Mark a notification as seen in the notification inbox
+     *
+     * @Route("/notification/seen/{notification}/{notifiable}", name="inbox_mark_seen")
+     *
+     * @param Notification        $notification
+     * @param NotifiableInterface $notifiable
+     *
+     * @throws \Doctrine\ORM\NonUniqueResultException
+     * @throws \Doctrine\ORM\EntityNotFoundException
+     * @throws \Exception
+     *
+     * @return Response
+     */
+    public function inboxMarkSeenAction($notification, $notifiable)
+    {
+        $this->notificationManager->markAsSeen(
+            $this->notificationManager->getNotifiableInterface($this->notificationManager->getNotifiableEntityById($notifiable)),
+            $this->notificationManager->getNotification($notification),
```

Comment:
You can store the two parameters to the markAsSeen function in a variable to keep the line shorter and more readable. And re-use this variable later when getting the list.